src/TheThingsNetwork.h

```diff
@@ -23,6 +23,19 @@
 #define HEX_CHAR_TO_NIBBLE(c) ((c >= 'A') ? (c - 'A' + 0x0A) : (c - '0'))
 #define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
 
+/* BEGIN TEST */
+
+#ifdef DEBUG
```

Comment:
So in order to use these in examples I need to do:

```c
#define DEBUG 1
```

*before* I include TheThingsNetwork?